src/app/(dashboard)/mypage/page.tsx

```diff
@@ -1,9 +1,11 @@
+import Password from '@/components/mypage/Password';
 import Profile from '@/components/mypage/Profile';
 
 export default function MyPage() {
   return (
     <div>
```

Comment:
여기 추후에 스타일 적용하실 생각이 아니시라면 프래그먼트 태그로 교체 제안드려요~

```suggestion
    <>
```